acceptance_tests/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
Where does that come from and what is it for? O.o

Reply:
Good catch. I guess that this was generated automatically when I was updating cypress while investigating the problem we had with the integration tests yesterday.
I will delete it